packages/scheduled-tasks/src/lib/ScheduledTaskHandler.ts

```diff
@@ -0,0 +1,66 @@
+import { container } from '@sapphire/framework';
+import type { Awaitable } from '@sapphire/utilities';
+import { ScheduledTaskRedisStrategy } from './strategies/ScheduledTaskRedisStrategy';
+import { ScheduledTaskStore } from './structures/ScheduledTaskStore';
+import type { ScheduledTaskBaseStrategy, ScheduledTasksOptions, ScheduledTasksTaskOptions } from './types';
+
+export class ScheduledTaskHandler {
+	public readonly strategy: ScheduledTaskBaseStrategy;
+
+	public constructor(options: ScheduledTasksOptions | undefined) {
+		this.strategy = options?.strategy ?? new ScheduledTaskRedisStrategy();
+	}
+
+	public async connect() {
+		await this.strategy.connect();
+	}
+
+	public create(task: string, payload: unknown, options?: ScheduledTasksTaskOptions | number) {
+		if (typeof options === 'number') {
+			options = {
+				type: 'default',
+				delay: options
+			};
+		}
+
+		return this.strategy.create(task, payload, options);
+	}
+
+	public createRepeated() {
+		const store = this._getStore();
+
+		return this.strategy.createRepeated(
+			store.repeatedTasks.map((piece) => ({
+				name: piece.name,
+				options: {
+					type: 'repeated',
+					...(piece.interval
+						? {
+								interval: piece.interval!
+						  }
+						: { cron: piece.cron! })
+				}
+			}))
+		);
+	}
+
+	public run(task: string, payload: unknown): Awaitable<unknown> {
+		const store = this._getStore();
+		const piece = store.get(task);
+
+		if (!piece) {
+			return;
+		}
+
+		return piece.run(payload);
+	}
+
+	private _getStore(): ScheduledTaskStore {
```

Comment:
```suggestion
	private getStore(): ScheduledTaskStore {
```

I don't really like `_` as a private prefix. We have the `private` keyword in TS so we don't need another specific indicator.

In fact, can this not become a getter? `private get store(): SecheduledTaskStore`. Not sure if that works.